OpenRA.Game/Settings.cs

```diff
@@ -450,10 +451,10 @@ public static string SanitizedPlayerName(string dirty)
 
 		static void LoadSectionYaml(MiniYaml yaml, object section)
 		{
-			var defaults = Activator.CreateInstance(section.GetType());
+			var defaults = Unforgiving.Activator.CreateInstance(section.GetType());
 			FieldLoader.InvalidValueAction = (s, t, f) =>
 			{
-				var ret = defaults.GetType().GetField(f).GetValue(defaults);
+				var ret = defaults.GetType().GetFieldUnforgiving(f).GetValue(defaults);
```

Comment:
Some examples above of the "unforgiving" class to give us non-null versions of existing reflection code.